Tests/ResponseTests.swift

```diff
@@ -518,7 +518,7 @@ final class ResponseTryMapTestCase: BaseTestCase {
         XCTAssertNil(response?.data)
         XCTAssertEqual(response?.result.isFailure, true)
         XCTAssertEqual(response?.error?.asAFError?.isSessionTaskError, true)
-        XCTAssertEqual((response?.error?.asAFError?.underlyingError as? URLError)?.code, .cannotFindHost)
+        XCTAssertTrue(urlHostErrorCodes.contains(response!.error!.asAFError!.urlErrorCode))
```

Comment:
This should be structured similar to the previous line, where we compare the value optionally.